inference-engine/src/transformations/src/transformations/common_optimizations/algebraic_simplification.cpp

```diff
@@ -234,6 +235,7 @@ static bool replace_transpose_with_reshape(shared_ptr<Node> transpose) {
 }
 
 bool pass::AlgebraicSimplification::run_on_function(shared_ptr<Function> f) {
+    MATCHER_SCOPE(AlgebraicSimplification_run_on_function);
```

Comment:
I think it should be TRANSFORMATION_SCOPE?

Reply:
TRANSFORMATION_SCOPE is disabled when we collect the statistic.

Reply:
I renamed the macro.